src/core/or/channel.c

```diff
@@ -2409,11 +2412,15 @@ channel_get_for_extend,(const char *rsa_id_digest,
       continue;
     }
 
+    const int matches_target =
```

Comment:
const bool instead?

Reply:
Returned bool and changed variable to bool in 066d2deb3d322e960600d80739739199d8c4cfa6.